### DISCUSSION TITLE
来绳网冒个头www

### DISCUSSION THREAD

**Discussion by Fukacis** [share121/inter-knot#3149] [其他]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >b站的讨论环境疑似是有点极端了，看看绳网缓解一下<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >话说有磕丹星的朋友吗？好奇探头<img src="https://img-static.mihoyo.com/communityweb/upload/460a345beb85c449f354a38e2cd8e01c.png" alt="佩拉 懵" width="75" >

**Comment by Nie7bai** [comment]
什么都磕只会害了你<img src="https://img-static.mihoyo.com/communityweb/upload/8f9225dcc57a028231f1d67fdab9135c.png" alt="三月七_骄傲" width="75" >

**Reply by Fukacis** [reply]
哇在别的帖子下面有看到你了，高强度活跃的绳网网友੭ ᐕ)੭*⁾⁾

**Reply by Nie7bai** [reply]
(灬╹ω╹灬)
